### PR TITLE
search: log failures of zoektIndexedRepos and searchFilesInRepo

### PR DESCRIPTION
**Pull request by ijt**
This is meant to help us track down #3294.

### CODE REVIEW

File: cmd/frontend/graphqlbackend/textsearch.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"gopkg.in/inconshreveable/log15.v2"
```
Comment (slimsag) [inline cmd/frontend/graphqlbackend/textsearch.go]: weird that this is in the middle of stdlib imports, no?

Comment (ijt) [reply cmd/frontend/graphqlbackend/textsearch.go]: Moved it.

**Comment by slimsag** [comment]
I can cherry pick this change into 3.2.1 which I am releasing in a few hours if you like

**Comment by ijt** [comment]
> I can cherry pick this change into 3.2.1 which I am releasing in a few hours if you like

Yes please.

**Comment by slimsag** [comment]
Cool, will do, I'll ping you via Slack when I ship that version.

**Comment by ijt** [comment]
> Cool, will do, I'll ping you via Slack when I ship that version.

Sounds good. Can you also ping @KattMingMing ?

**Pull request merged**